test/test.js

```diff
@@ -309,3 +312,15 @@ test('All parameters are optional', async t => {
   inspector = new Inspector()
   t.ok(inspector)
 })
+
+test('Can create a heapdump', async t => {
+  t.teardown(teardown)
+  const dir = await getTmpDir(t)
+  inspector = new Inspector({ inspector: nodeInspector })
+
+  const loc = path.join(dir, 'heap.heapsnapshot')
+  t.absent(fs.existsSync(loc), 'sanity check')
+
+  await inspector.writeHeapSnapshot(loc)
+  t.ok(fs.existsSync(loc), 'Heapdump written')
```

Comment:
Should probably clean up after and remove the file, otherwise the `t.absent` above will fail.

Reply:
`getTmpDir` handles that for us, it ensures that the dir is empty (https://github.com/mafintosh/test-tmp/blob/2fb11b1f990085e6a86a38420a3f0cc295ad155d/index.js#L12-L14) and if the brittle `t` object is passed in it cleans up after itself (https://github.com/mafintosh/test-tmp/blob/2fb11b1f990085e6a86a38420a3f0cc295ad155d/index.js#L18)